src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -184,209 +118,225 @@ function SubmissionsFactory(
       workerPool = []
       // Creates a new AbortController for every request
       downloadAbortController = new AbortController()
+      const { formId, startDate, endDate } = params
```

Comment:
does this need to be destructured?

Reply:
thanks for pointing this out! i preferred this method because i wanted to be explicit about what we were passing into the newly written ts function; however, there were some other functions (eg: `generateDownloadUrl`) that rightfully expect a dict and hence, i did it this way rather than passing in `params` to the ts service